README.md

```diff
@@ -306,6 +306,7 @@ var fs = new Filer.FileSystem();
 * [fs.mknod(path, mode, callback)](#mknod)
 * [fs.rmdir(path, callback)](#rmdir)
 * [fs.mkdir(path, [mode], callback)](#mkdir)
+* [fs.mkdtemp(path, [mode], callback)](#mkdtemp)
```

Comment:
Here you use `[mode]` but below you refer to `options`.  Can you fix one or the other?